### DISCUSSION TITLE
Build stuck in queue for the past 40+ minutes

### DISCUSSION THREAD

**Discussion by romdotdog** [vercel/vercel#6253] [Help]
Hi! I'm looking to deploy a basic Node.js + React project. My first two builds succeeded but henceforth my builds get stuck in the queue. I have no other projects, and nothing else building, so it couldn't be that I've reached the concurrent builds limit. I've tried deleting and restarting the project to no avail. Any help would be appreciated, thanks!

**Comment by romdotdog** [comment]
After 2 hours and 15 minutes I can now finally build my project.

**Comment by fu-sen** [comment]
This has been a reporter for the last few days. I also experienced it.
https://github.com/vercel/vercel/discussions/4948

**Comment by molimat** [comment]
Are you guys experiencing timeouts on your servless func?

**Reply by fu-sen** [reply]
No, in my case it was also experienced on a simple Static Site.
